content/kips/kip-18.md

```diff
@@ -0,0 +1,69 @@
+---
+kip: 18
+title: Cross Margin
+status: Draft
+author: Jeremy Chiaramonte
+created: 2022-06-30
+---
+
+## Summary
+
+Cross margin enables futures positions to be managed from a singular margin account.
+
+## Abstract
+
+An abstraction layer to be built on top of the Synthetix Futures (V1) protocol. It enables the creation of "margin accounts" for each trader. Margin accounts enable Kwenta to provide cross margin functionality.
+
+## Motivation
+
+The current Futures system requires traders to deposit margin into specific markets before trading. This two step process causes friction when trying to act on market conditions and when managing margin between multiple markets. Cross margin improves the UX of trading on Kwenta.
+
+## Specification
+
+A major change introduced with this KIP is the introduction of a "margin account" system. Each trader will be required to set up a smart contract wallet (this step is managed by the UI) that holds margin on behalf of the trader. Smart contract accounts also open the door for future functionality as seen by [KIP-19: Advanced Orders](./kip-19.md).
+
+Cross margin is achieved by bundling together the margin step and the trade step when modifying a futures position. With the current implementation a trader can also open/modify/close multiple positions at once (atomically). It will be up to the frontend implementation on how to manage and rebalance margin. 
+
+### Rationale & User Flows
+
+- As a **trader** I want:
+    - To gain exposure to a **singular asset**:
+        - By opening BTC position without having to perform two transactions.
+        - By opening BTC position using the margin from closing a LINK position.
+    - To gain exposure to **multiple assets**:
+        - By opening BTC and ETH position without having to perform multiple transactions.
+    - To gain exposure on the **negative correlation of two assets**:
```

Comment:
Similar to the other comment, I think "negative correlation" might be too specific. Could say something like "To speculate on the ratio between two asset prices"